containers/create-proxy-config.py

```diff
@@ -1,10 +1,34 @@
 #!/usr/bin/python3
 
+import argparse
 import glob
 import os
 import shutil
 import socket
 
+def parse_arguments():
+  description = 'Creates a configuration directory for a Proxy container'
+  parser = argparse.ArgumentParser(description=description)
+  parser.add_argument('fqdn', metavar='FQDN', type=str, nargs=1, help='FQDN of the Proxy')
+  parser.add_argument("-c", "--country", dest="country", help="Country for auto-generation of SSL certificate", default="US")
+  parser.add_argument("-s", "--state", dest="state", help="State for auto-generation of SSL certificate", default="STATE")
+  parser.add_argument("-i", "--city", dest="city", help="City for auto-generation of SSL certificate", default="CITY")
+  parser.add_argument("-o", "--organization", dest="organization", help="Organization for auto-generation of SSL certificate", default="ORGANIZATION")
+  parser.add_argument("-u", "--organization-unit", dest="organization_unit", help="Organization unit for auto-generation of SSL certificate", default="ORGANIZATION UNIT")
+  parser.add_argument("-e", "--email", dest="email", help="E-mail for auto-generation of SSL certificate", default="name@example.com")
+  results = parser.parse_args()
+  return results
+
+args = parse_arguments()
+fqdn = args.fqdn[0]
+country = args.country
+state = args.state
+city = args.city
+organization = args.organization
+organization_unit = args.organization_unit
+email = args.email
+cname = '.'.join(fqdn.split('.')[-2:])
```

Comment:
I think this should be the following. FQDN can be specified with a trailing dot for the dns root, would give you canonical names such as `com.` instead of `example.com` for `this.example.com.`
```suggestion
cname = '.'.join(fqdn.rstrip(".").split('.')[-2:])
```